### ISSUE TITLE
FEATURE: Move docs to GitHub Wiki

### ISSUE THREAD

**Issue by RealPacket** [face-hh/webx#194]
**Is your feature request related to a problem? Please describe.**
GitBook always renders in light theme, can't find a dark mode toggle either.

**Describe the solution you'd like**
Move docs to GitHub wikis.
You may need to enable them in the settings of your repo for them to show in the menu bar.

**Describe alternatives you've considered**
Drinking oil
Pouring oil on myself
Not doing the above
Sending a pipe bomb to the GitBook engineer that made the light theme default
Adopting a black cat to replace the white parts of the page with a physical black cat (and to pet).

**Additional context**
<details><summary>GitBook page displaying theme as flashbang theme</summary>
<img src="https://github.com/face-hh/webx/assets/107498533/b5d74511-2635-495c-8e62-8771e3c6234b alt="flashbang"/>
</details>

**Comment by GStudiosX2** [comment]
Gitbook has a dark theme face just has to enable it

**Comment by RealPacket** [comment]
Yes, but it would be better if the docs would be in the same location as the repository.

**Comment by GStudiosX2** [comment]
> Yes, but it would be better if the docs would be in the same location as the repository.

The docs are in the repo though 
https://github.com/face-hh/webx/tree/master/docs

**Comment by RealPacket** [comment]
But it would make more sense being in the Wiki.
